src/roles.ts

```diff
@@ -0,0 +1,6 @@
+export const rolesIds = {
+  leadAdmin: '870429095941517332',
+  admins: '870411279108571216',
+  leadDev: '870435957969657886',
+  leadAmbassador: '870501520796418059'
+}
```

Comment:
Great idea!

Reply:
We can create some config files for prod/dev environments. So we don't need to all the time change ID on environment change. There we can keep all the IDs, and store all .env properties. In this case, we will have all the important stuff in one place. 

For example .env: 
instead to use process.env.SOMETHING in multiple files you can initiate in the config file as:
export const something = process.env.SOMETHING
If you understand what I mean. In the end, we can save some time on switching from dev to prod.

Reply:
Yeah I think a config file would help us out